### ISSUE TITLE
It is actually a question.

### ISSUE THREAD

**Issue by NewtronReal** [al-caughey/Google-Meet-Attendance#102]
will this work even the tab is inactive??

**Comment by Eagle-Man** [comment]
Yes.  This will still record attendance as long as your Meet window is still open somewhere, even if it's not the tab in focus.  From the participant end, again, as long as their Meet tab is still open, they will be seen as present.  The only exception is if a participant is using their mobile device, they will not be seen as present if they switch apps.
